index.html

```diff
@@ -1,48 +1,297 @@
 
-<ul class="connect" >
-    <li class="media"><a href="#"><img alt="twitter" src="twitter.png"></a></li>
-    <li class="media"><a href="#"><img alt="linkedin" src="linkedin.png"></a></li>
-    <li class="media"><a href="#"><img alt="medium" src="medium.png"></a></li>
-    <li class="media"><a href="#"><img alt="github" src="github.png"></a></li>
-    <li class="media"><a href="#"><img alt="angellist" src="angellist.png"></a></li>
-</ul>
+            <ul class="connect" >
+                <li id="twitter" class="media"><a href="#"><img alt="twitter" src="img/twitter.png"></a></li>
+                <li class="media"><a href="#"><img alt="linkedin" src="img/linkedin.png"></a></li>
+                <li class="media"><a href="#"><img alt="medium" src="img/medium.png"></a></li>
+                <li class="media"><a href="#"><img alt="github" src="img/github.png"></a></li>
+                <li class="media"><a href="#"><img alt="angellist" src="img/angellist.png"></a></li>
+            </ul>
 
 
- </section>
- </main>
+        </section>
+
+    
+    
+    </main>
+
  
     
-</body>
+        <header>
+
+            <div><button  class="scroll-down" type="button">
+                <i class="fa fa-angle-down fa-2x"></i>
+            </button></div>
+
+            <div class="recent-works"><h1>My Recent Works</h1></div>
+            <div class="indicator">
+                <img alt="indcator" src="img/Indicator.png">
+            </div>       
+
+        </header>
+    <div class="wrapper">
+
+        <div>
+            <section id="works1" class="container3">
+
+
+                <div class="item3"><img alt="work1" src="img/work1.png"></div>
+
+                
+                            
+                <div class="item3"><h2>Multi-Post Stories<br> Gain+Glory</h2></div>
+
+
+                <div class=" item3 btn-group">
+                    <button class="languages"> Ruby on rails</button>
+                    <button class="languages"> css</button>
+                    <button class="languages"> JavaScript</button>
+                    <button class="languages"> html</button>
+                </div>
+
+                
+                    <div class="item3">
+                        <button class="see-project">See project</button></div>
+                
+                
+
+
+
+            </section>
+        </div>
+
+        <div>
+            <section id="works2" class="container4">
+
+
+                <div class="item4"><img alt="work2" src="img/work2.png"></div>
+
+                
+                            
+                <div class="item4"><h2>Multi-Post Stories<br> Gain+Glory</h2></div>
+
+
+                <div class=" item4 btn-group">
+                    <button class="languages"> Ruby on rails</button>
+                    <button class="languages"> css</button>
+                    <button class="languages"> JavaScript</button>
+                    <button class="languages"> html</button>
+                </div>
```

Comment:
- [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list ul and each of them a li, let us wrap the technologies with the `ul` and `li`